docs/ht_ww.md

```diff
@@ -1,9 +1,53 @@
 # OVOS Listener - WakeWords / HotWords
-OVOS uses "wakewords" to activate the system.  This is what "hey Google" or "Alexa" is on proprietary devices.  By default, OVOS uses the WakeWord "hey Mycroft".
+OVOS uses "wakewords" to activate the system. This is what "hey Google" or "Alexa" is on proprietary devices. By default, OVOS uses the WakeWord "hey Mycroft".
+
+OVOS "hotwords" is the configuration section in `mycroft.conf` config  to specify what the WakeWord do.  Multiple "hotwords" can be used to do a variety of things from putting OVOS into active listening mode, a WakeWord like "hey Mycroft", to issuing a command such as "stop" or "wake up"
+
+Each hotword can do one or more of the following (with example configs):
+* trigger listening, also called a wake_word (default option)
+  * `“Listen” : “True”`
+
+* play a sound (only or with other action), you can set a specific sound per hotword.
+  * `"sound": "snd/start_listening.wav"`
```

Comment:
Consider mentioning that taking another action would require a plugin or skill to act on the bus message that is triggered by the WakeWord.

Reply:
I think leaving out the part about other actions (instead of only playing a sound) completely is maybe more clear, because the other lines that follow the "sound" config make that more clear (imho).

Reply:
@JarbasAl When I add the "utterance" config to the HotWord section AND "sound" to the hotword, the custom sound is not played. 
So like this:

```
 "hotwords": {
    "hotkey": {
        "module": "ovos_ww_hotkeys",
        "listen": false,
        "active": true,
        "hotkey": "space",
        "utterance": "what time is it?",
        "sound": "snd/acknowledge.mp3"
      }
  }
```
Is that a possible issue or by design?